jax/_src/pjit.py

```diff
@@ -1385,7 +1386,9 @@ def _pjit_lower_cached(
     name: str,
     in_is_global: Sequence[bool],
     keep_unused: bool,
-    always_lower: bool):
+    *,
+    always_lower: bool,
+    lowering_platform: Optional[str]):
```

Comment:
Just default to None here so that you don't have to thread it from line 1293 too?

Reply:
I prefer to not have defaults because those can hide bugs. If there is no default value, it forces you to think what to do at each call site, and there aren't that many. I would rather not miss one. As you can see there are only a handful of places where I need to pass `None`

Reply:
Ok, sounds good. There is one more place you are missing in pjit then in _pjit_partial_eval.

Reply:
Fixed the pjit_partial_eval, thanks!